src/components/popup/PopupDirective.js

```diff
@@ -11,12 +11,22 @@ goog.require('ga_print_service');
   ]);
 
   module.directive('gaPopup',
-    function($rootScope, $translate, $window, gaBrowserSniffer,
-        gaPrint) {
-      var zIndex = 2000;
+    function($rootScope, $translate, $window, gaBrowserSniffer, gaPrint) {
+      var zIndex;
+      var screenLimit = 0;
+      var screenTabletLimit = 768;
+      var screenMobileLimit = 480;
+
```

Comment:
hmm. same values as in css. when changed there, we have to change here as well.

No other way that is more maintainable?